main/RFXNames.cpp

```diff
@@ -275,6 +275,7 @@ static const STR_TABLE_SINGLE HardwareTypeTable[] = {
 	{ HTYPE_OctoPrint, "OctoPrint (MQTT/Gina Haussge) with LAN interface", "OctoPrint" },
 	{ HTYPE_Meteorologisk, "Meteorologisk institutt Norway (Weather Lookup)", "Meteorologisk" },
 	{ HTYPE_AirconWithMe, "AirconWithMe Wifi Airco module", "AirconWithMe" },
+	{ HTYPE_VirtualThermostat, "Virtual Thermostat"							,       "VirtualThermostat" },
```

Comment:
Please check the indent, it is a space, not 100 spaces ;)

Reply:
ok del tab